docs/install.rst

```diff
@@ -68,6 +68,16 @@ Installing prerequisites
 
       If Yosys 0.9 (or newer) is not available, `build Yosys from source`_.
 
+   .. platform-choice:: arch
+      :altname: linux
+      :title: Arch Linux
+
+      Install GTKWave and Yosys by running:
+
+      .. code-block:: console
+
+         $ sudo pacman -S gtkwave yosys
```

Comment:
I think this should include Python (and pip) in case the reader wants to install an unreleased version of nMigen.

Reply:
Ah, right.